src/utils/fetchers.ts

```diff
@@ -34,27 +67,39 @@ export const loadCachedExplore = async (
   )
 }
 
-export const loadUsers = async (
-  coreSDK: LookerSDK,
-  ids: number[],
-) => {
-  // @ts-ignore
-  return coreSDK.ok(coreSDK.all_users({ids: ids.toString() }))
+export const loadUsers = async (coreSDK: ILooker40SDK, ids: string[]) => {
+  if (ids && ids.length > 0) {
+    return coreSDK.ok(
+      // TODO extensionSDK cannot handle serialization of DelimArray
+      coreSDK.all_users({ ids: ids.join(',') as any })
+    )
+  }
+  return []
 }
 
-export const loadAllModels = async (sdk: LookerSDK) => {
-  return loadCached("all_lookml_models", () => sdk.ok(sdk.all_lookml_models()))
+export const loadAllModels = async (sdk: ILooker40SDK) => {
+  return loadCached('all_lookml_models', () =>
+    sdk.ok(sdk.all_lookml_models({}))
+  )
 }
 
-export const getMyUser = async (sdk: LookerSDK) => {
+export const getMyUser = async (sdk: ILooker40SDK) => {
```

Comment:
definitely an option in another PR to make this functional

Reply:
Agreed. More love is needed here